types/haxball-headless-browser/index.d.ts

```diff
@@ -54,9 +54,9 @@ declare class RoomObject {
 
     stopRecording(): Uint8Array;
 
-    setPassword(pass: string): void;
+    setPassword(pass?: string): void;
```

Comment:
The documentation says

> `setPassword(pass : string) : void`
>
> Changes the password of the room, if pass is null the password will be cleared.

I guess we can’t look at the source code to validate, but from the text, I would type this as

```suggestion
    setPassword(pass: string | null): void;
```

Reply:
Thats correct, but the actual behavior is, if the argument is empty, it also removes the password (i've tested it manually and there is no way to cover this *empty* password check, as the application doesn't have a function to GET the current password). What do you think?

Reply:
I think it should at least be `setPassword(pass?: string | null)` to allow `null` since the docs explicitly mention it, but I feel like `setPassword(pass: string | null)` (non-optional param) makes for a better API, since `setPassword(null)` is more understandable than `setPassword()`.